### PR TITLE
FACT-491 amending mobile issues, changing to use summary lists

### PR DESCRIPTION
**Pull request by amstevenson**
### JIRA link ###
https://tools.hmcts.net/jira/browse/FACT-491

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/views/court-details/in-person-court.njk [hunk 1]
```diff
@@ -91,50 +102,49 @@
...
-                {% endif %}
-              </div>
-            </div>
-          </div>
-          {% endfor %}
+            <dl class="govuk-summary-list govuk-summary-list--no-border">
+              {% for telephone in results.contacts %}
+                <div class="govuk-summary-list__row">
+                  <dd class="govuk-summary-list__key">
+                      {# <p class="govuk-body-m">
```
Comment (KianKwa) [inline src/main/views/court-details/in-person-court.njk:133]: Think there is a slight indentation issue here :)

Comment (KianKwa) [reply src/main/views/court-details/in-person-court.njk:133]: Actually is this commented out? If so they can be removed

File: src/main/views/court-details/in-person-court.njk [hunk 2]
```diff
@@ -91,50 +102,49 @@
...
+                        {{ telephone.description}}
+                      </p> #}
+
+                    {{ telephone.description}}
```
Comment (KianKwa) [inline src/main/views/court-details/in-person-court.njk:137]: Slight formatting issue. No space after the word description

**Review by KianKwa — APPROVED** [review]

**Comment by Thomas-Geraghty** [comment]
Merged in master to fix sec vuln blocking build

**Pull request merged**